src/main/java/io/icker/factions/api/persistents/User.java

```diff
@@ -146,4 +156,40 @@ public static void save() {
         Database.save(User.class, STORE.values().stream().toList());
     }
 
+    //region Power Management
+    public static int getMaxPower(final @NotNull UUID userUUID) {
+        final var user = get(userUUID);
+        return user != null ? user.getMaxPower() : 0;
+    }
+
+    public int getMaxPower() {
+        return CONFIG.POWER.MEMBER + getPermissionPower(getID());
+    }
+
+    public static int getPower(final @NotNull UUID userUUID) {
+        return get(userUUID).getPower();
+    }
+
+    public int getPower() {
+        return power;
+    }
+
+    @SuppressWarnings("unused") //util
+    public static int setPower(final @NotNull UUID userUUID, final int newPower) {
+        return get(userUUID).setPower(newPower);
+    }
+
+    public int setPower(final int newPower) {
+        return power = newPower;
```

Comment:
Just curious, why did you return the new value here?

Reply:
Because this allows for better inlining in the occasion someone wants to make an addon for the mod.

For example, if I want to make a potion that gives me  X power and then splits that power between my teammates, I can just do `int toTeam = u.setPower(x)/u.getFaction().getUsers().length()` instead of juggling variabiles; I admit it's a niche use case, but I coded that in out of habit.